client/src/components/TripCard/TripCard.css

```diff
@@ -0,0 +1,30 @@
+.TripCard {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  width: 100%;
+  height: 100%;
```

Comment:
Change from percentage to rem. Talk to @shelbyvjacobs if you have any questions!

Reply:
I fixed all the percentages except the ones you highlighted, that div needs to be 100% of its parent, so the bug @shelbyvjacobs was talking about won't affect it.